### PR TITLE
Implement `actionable_blocked_uri?` checks

### PR DESCRIPTION
**Pull request by jacobbednarz**
Lays the groundwork for filtering out unactionable violation reports. The idea
here is that not every violation report you receive is actually going to be
something you have control over so you just drop it on the ground and not clog
up your reporting tooling with it.

This implements the first lot of checks based on the `blocked-uri` key of the
report. This will check against a list of known unactionable URIs and halt the
job should one be encountered. The list is compiled from all over the web but
notable mentions to report-uri.io and @oreoshake's [blog article about
Twitter's CSP collector design](https://oreoshake.github.io/csp/twitter/2014/07/25/twitters-csp-report-collector-design.html).

### CODE REVIEW

File: jobs/create_csp_violation_job.rb [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+    return unless actionable_blocked_uri?
+  end
+
+  # Private: Determine if we have an actionable report based on 'blocked-uri'.
+  #
+  # There are some blocked URIs that we just cannot do anything with. These
+  # are generated by things like browser extensions, internal browser pages,
+  # MITM proxies and are outside of our control.
+  def self.actionable_blocked_uri?
+    !KNOWN_UNACTIONABLE_BLOCKED_URIS.include?(@violation_report['blocked-uri'])
```
Comment (oreoshake) [inline jobs/create_csp_violation_job.rb]: You'd need to parse out the scheme or treat `KNOWN_UNACTIONABLE_BLOCKED_URIS` as a set of regexen, right? IIRC there were things after e.g. `safari-resource://kdjerhrgkljhdekjvhskjehfkuhe`

Comment (jacobbednarz) [inline jobs/create_csp_violation_job.rb]: Yep - Good catch! I had this around the wrong way in my head. I've just pushed up a change to use `start_with?` and the known unactionable URI's via 3d898d4. This should now work as intended.

File: jobs/create_csp_violation_job.rb [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+    return unless actionable_blocked_uri?
+  end
+
+  # Private: Determine if we have an actionable report based on 'blocked-uri'.
+  #
+  # There are some blocked URIs that we just cannot do anything with. These
+  # are generated by things like browser extensions, internal browser pages,
+  # MITM proxies and are outside of our control.
+  def self.actionable_blocked_uri?
+    !@violation_report['blocked-uri'].start_with?(
```
Comment (oreoshake) [inline jobs/create_csp_violation_job.rb:20]: :cool: TIL `start_with?` accepts varargs.

Comment (jacobbednarz) [inline jobs/create_csp_violation_job.rb:20]: yep! i think this is nicer than a giant, messy regex.

**Pull request merged**